### ISSUE TITLE
Update documentation

### ISSUE THREAD

**Issue by lucventurini** [EI-CoreBioinformatics/mikado#136]
The documentation for Mikado might need some refreshing in some areas, to clarify points and update on the latest developments. This should be completed before we release the next version.

**Comment by lucventurini** [comment]
It is **essential** to make sure that the documentation explicitly explains that Mikado serialise must use ORFs **called on the Mikado prepare output**, not on the input transcript assemblies. Otherwise fatal mistakes can happen, see #206.

**Comment by lucventurini** [comment]
Hi @ljyanesm , I have updated the documentation and the comments for the configuration files in the `issue-136` branch.

This branch is strictly for modifying the documentation, comments of the configuration files and, at most, solving bugs related to either of the conditions above (hence my faffing around with the configuration this morning - however, the modifications do not modify the running of Mikado *per se*).

**Comment by ljyanesm** [comment]
Hi @lucventurini, I can't see the branch you mention. Could it be in a different fork of this project?

**Comment by lucventurini** [comment]
Hi @ljyanesm,
                         you are right, it's on my personal fork. I just pushed it here as well.

**Comment by lucventurini** [comment]
Update: with 2f5a6b870063a7196346e5e8f0e09d08220d85ce, the `issue-136` branch is now current with all the rest of the modifications.

@swarbred , as discussed by chat, on this branch I have also introduced the ability of specifiying per-sample stripping of the CDS during prepare and per-sample protection from the chimera splitting.

I have revised the documentation for the configuration and the algorithms, working my way through the other stages of the pipeline.
